### ISSUE TITLE
test model with different input image size

### ISSUE THREAD

**Issue by kiashann** [subpic/koniq#8]
Hi, It's possible to test the model with different input sizes? I saw the 512 * 384 size in the code
What is the range of score that model can be predict? 
@subpic

**Comment by subpic** [comment]
Yes, it should be possible, the model allows other input shapes as long as they have 3 channels. The output range is related to the range used for the quality scores during training. I believe that was [1,100].

**Comment by kiashann** [comment]
For one image I got a score above 100. Why was it happen? What's the problem?

**Comment by subpic** [comment]
Even though the training scores are bounded, the predictions are not constrained to [1,100]. It's possible that for some images the predictions will be outside those bounds.
